test/models/work_test.rb

```diff
@@ -0,0 +1,115 @@
+require "test_helper"
+require 'pry'
+
+describe Work do
+  describe 'relations' do
+    it 'has many votes' do
+      work = works(:nemo)
+
+      work.votes.length.must_equal 3
+      work.votes[0].must_equal votes(:one)
```

Comment:
do we want to check the order of what comes into the `votes` array? I'm not sure that it's important that we do. However, if you wanted to do this, I might instead check with
```ruby
work.votes.include?( votes(:one) ).must_equal true
```